### PR TITLE
feat: Update baggageItemToTag filter name

### PR DESCRIPTION
**Pull request by addityasingh**
Update `baggageItemToTag` filter name to `tracingBaggageToTag `. See this PR for details https://github.com/zalando/skipper/pull/1160

### CODE REVIEW

**Comment by ruiaraujo** [comment]
:+1:

File: docs/reference/filters.md [hunk 1]
```diff
@@ -1511,7 +1511,7 @@ It is also possible to enable this behavior centrally for a Skipper instance wit
 the -rfc-patch-path flag. See
 [URI standards interpretation](../../operation/operation/#uri-standards-interpretation).
 
-## bearerinjector
+## bearerInjector
```
Comment (aryszka) [inline docs/reference/filters.md]: this was actually cased like that, not saying that it's the right way ;)
https://github.com/zalando/skipper/blob/master/filters/auth/bearer.go#L9

**Comment by aryszka** [comment]
:+1:

File: docs/reference/filters.md (hunk 1 above)
Comment (addityasingh) [reply docs/reference/filters.md]: In that case, I will revert this change from current PR as its not relevant. I will create an issue to discuss if its possible to change the name by keeping it backward compatible or would it be a breaking change

Comment (addityasingh) [inline docs/reference/filters.md]:
```suggestion
## bearerinjector
```

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**